src/AasxServerBlazor/Properties/launchSettings.json

```diff
@@ -10,7 +10,7 @@
     },
     "AasxServerBlazor": {
       "commandName": "Project",
-      "commandLineArgs": "--no-security --secret-string-api 1234 --aasx-in-memory 1000 --data-path \"\\\\wsl.localhost\\Ubuntu\\home\\oze\\plugfest\\aasxs-view\" --edit --external-blazor http://localhost:5001",
+      "commandLineArgs": "--no-security --save-temp 5 --external-blazor http://127.0.0.1:5001 --edit --data-path \"C:\\Users\\Y97CO3\\OneDrive - PHOENIX CONTACT GmbH & Co. KG\\04_Abteilung_DI\\02_Projekte\\02_Aasx_Server\\01_AASX-Server_AASen\\aasx_bachelor\\aasx_10\" --start-index 1000 --with-db",

```

Comment:
@blidhpxc I would revert the path. It is a public repo and one might get to much information of your local machine